system-tests/lib/cre/contracts/contracts.go

```diff
@@ -62,6 +63,20 @@ type dons struct {
 	c map[string]donConfig
 }
 
+func (d *dons) donsOrderedByID() []donConfig {
+	out := make([]donConfig, 0, len(d.c))
+	for _, don := range d.c {
+		out = append(out, don)
+	}
+
+	// Use sort library to sort by ID
+	sort.Slice(out, func(i, j int) bool {
+		return out[i].id < out[j].id
+	})
```

Comment:
Could simplify with newer std lib:
```suggestion
	out := slices.Collect(maps.Values(d.c))

	// Use sort library to sort by ID
	slices.SortFunc(out, func(a, b donConfig) bool { return a.id < b.id })
```

Reply:
Simpler alternative below :point_down: